doc/branch.rst

```diff
@@ -85,47 +81,139 @@ as a branch if one of its choices is excluded::
 Because the ``else`` clause is excluded, the ``if`` only has one possible next
 line, so it isn't considered a branch at all.
 
-
 Structurally partial branches
 -----------------------------
 
 Sometimes branching constructs are used in unusual ways that don't actually
-branch.  For example::
+branch. For example::
 
     while True:
         if cond:
             break
         do_something()
 
-Here the while loop will never exit normally, so it doesn't take both of its
-"possible" branches.  For some of these constructs, such as "while True:" and
-"if 0:", coverage.py understands what is going on.  In these cases, the line
-will not be marked as a partial branch.
+Here the ``while`` loop will never exit normally, so it doesn't take both of
+its "possible" branches. For some of these constructs, such as ``while True:``
+and ``if 0:``, coverage.py understands what is going on.
```

Comment:
Again, I feel like something important is lost by dropping the last sentence of the paragraph.